utils/misc/dbutils.go

```diff
@@ -79,9 +79,9 @@ func NewDatabaseConnectionPool(
 	maxConnLifetime := maxConnLifetimeVar.Load()
 	db.SetConnMaxLifetime(maxConnLifetime)
 
-	ticker := time.NewTicker(5 * time.Second)
-	defer ticker.Stop()
 	rruntime.Go(func() {
+		ticker := time.NewTicker(5 * time.Second)
```

Comment:
Instead of hardcoding it, can we make it configurable?
```suggestion
		ticker := time.NewTicker(conf.GetDuration("db.pool.configUpdateInterval", 1, time.Second))
```